### ISSUE TITLE
RancherDesktop can not be installed

### ISSUE THREAD

**Issue by hohwille** [devonfw/ide#770]
### Expected behavior

`devon docker setup` should simply work

### Actual behavior

It does not even manage to download RancherDesktop.

### Steps to reproduce (bug) / Use Case of feature request (enhancement)

```
devon docker setup
Rancher Desktop set as option
Starting installation of rancher in version 1.3.0 to /d/projects/mmm/updates/install/rancherdesktop

******** ATTENTION ********
ERROR: Download of rancher in version 1.3.0 failed after trying 0 mirror(s).
We are sorry for the inconvenience. Please check the above errors, resolve them and try again.
```

### Related/Dependent Issues

#690 

### Comments/Hints:

Check mirrors

### Affected version:

* 2022.04.001
* OS: tested on Windows, most probably all plattforms

**Comment by hohwille** [comment]
Download now works but next error is:
```
devon docker setup
Rancher Desktop set as option
Starting installation of rancher in version 1.3.0 to /d/projects/devonfw/updates/install/rancherdesktop
Trying to download rancher-1.3.0-windows.exe from https://github.com/rancher-sandbox/rancher-desktop/releases/download/v1.3.0/Rancher.Desktop.Setup.1.3.0.exe
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  365M  100  365M    0     0   579k      0  0:10:45  0:10:45 --:--:--  831k
Successfully extracted/moved archive rancher-1.3.0-windows.exe to updates/extracted
Successfully installed (installation software for) rancher
Installing Rancher Desktop for Windows...
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 7315k  100 7315k    0     0   738k      0  0:00:09  0:00:09 --:--:--  802k
between mv and pushd

wsl-vpnkit v0.3.2
This distro is only intended for running wsl-vpnkit.

Run the following commands from Windows or other WSL 2 distros to use.

    wsl.exe -d wsl-vpnkit service wsl-vpnkit start
    wsl.exe -d wsl-vpnkit service wsl-vpnkit stop

The following file will be copied if it does not already exist.

    /mnt/c/Users/hohwille/wsl-vpnkit/wsl-gvproxy.exe

Logs for wsl-vpnkit can be viewed here.

    wsl.exe -d wsl-vpnkit tail -f /var/log/wsl-vpnkit.log

Config for wsl-vpnkit can be edited here.

    /mnt/c/Users/hohwille/wsl-vpnkit/wsl-vpnkit.conf

Run the following command to see the default values.

    wsl.exe -d wsl-vpnkit cat /app/defaults.conf

Press [enter] key to continue...


/bin/sh: bash: not found
```

I cannot tell where this `/bin/sh: bash: not found` comes from but it seems the installation process is still not working correct.
Also I am confused what the log-line `between mv and pushd` should tell me.
Further, from the log output I can not tell if the error occurred whilst installing `vpnkit` or if that succeeded and the error happened afterwards while installing RancherDesktop.

**Comment by hohwille** [comment]
Also I tested the story #710 that was implemented with PR #760 
I manually called the following command
```
wsl.exe -d wsl-vpnkit service wsl-vpnkit start
```
It succeeded but gave me no output at all.
I also verified that the WSL container is present:
```
PS C:\Users\hohwille> wsl --list
Windows Subsystem for Linux Distributions:
rancher-desktop-data (Default)
rancher-desktop
wsl-vpnkit
```


Then I launched RancherDesktop while I was connected with VPN via Cisco AnyConnect and got the same error as always:
<img width="574" alt="RancherDesktop-VPN-issue" src="https://user-images.githubusercontent.com/1923119/170968177-7b82808d-e916-4967-bfab-4aa1ab6badb7.png">

Further observations:
```
wsl.exe -d wsl-vpnkit cat /app/defaults.conf
cat: can't open 'C:/Program Files/Git/app/defaults.conf': No such file or directory
 wsl.exe -d wsl-vpnkit tail -f /var/log/wsl-vpnkit.log
tail: can't open 'C:/Program Files/Git/var/log/wsl-vpnkit.log': No such file or directory
tail: no files
```

The above commands work in powershell or CMD but as I ran the setup in git-bash and got this logged as output and was even forced to confirm this, I think users will be slightly confused by all this.
